dbfit-java/build.gradle

```diff
@@ -83,7 +83,10 @@ subprojects {
         include('**/*Support*')
     }
 
-    task integrationTest(type: Test) {
+    task dbConnectionProperties {
+    }
+
+    task integrationTest(type: Test, dependsOn: dbConnectionProperties) {
```

Comment:
Good point. The sets of properties can be split into different parents / containers.